Documentation/contributing/development/images.rst

```diff
@@ -198,85 +198,49 @@ Image dependency:
 Update cilium-builder and cilium-runtime images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Login to quay.io with your credentials to the repository that you want to
-update:
+1. cilium-builder depends on cilium-runtime so one needs to update
+   cilium-runtime first. Steps 4 and 7 will fetch the digest of the image built
+   by GitHub actions.
 
-* `cilium-builder <https://quay.io/repository/cilium/cilium-builder?tab=builds>`__ - contains Cilium build-time dependencies
-* `cilium-runtime <https://quay.io/repository/cilium/cilium-runtime?tab=builds>`__ - contains Cilium run-time dependencies
+.. code:: bash
 
-0. After login, select the tab "builds" on the left menu.
+    $ make -C images/ update-runtime-image
 
-.. image:: ../../images/cilium-quayio-tag-0.png
-    :align: center
+2. Commit your changes and create a PR in cilium/cilium.
 
-1. Click on the wheel.
-2. Enable the trigger for that build trigger.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-1.png
-    :align: center
+    $ git commit -s images/ -m "update cilium-{runtime,builder}"
 
-3. Confirm that you want to enable the trigger.
+3. Ping one of the members of `team/build <https://github.com/orgs/cilium/teams/build/members>`_
+   to approve the build that was created by GitHub Actions `here <https://github.com/cilium/cilium/actions?query=workflow:%22Base+Image+Release+Build%22>`_.
 
-.. image:: ../../images/cilium-quayio-tag-2.png
-    :align: center
+4. Wait for the build to complete. Only after the image is available run:
 
-4. After enabling the trigger, click again on the wheel.
-5. And click on "Run Trigger Now".
+    $ make -C images/ update-runtime-image update-builder-image
 
-.. image:: ../../images/cilium-quayio-tag-3.png
-    :align: center
+5. Commit your changes and re-push to the PR in cilium/cilium.
 
-6. A new pop-up will appear to select your desired branch.
-7. If you're interested in simply bumping the image to have the latest
-   packages, then select the release branch (i.e. v1.7, v1.8). If you already
-   have a branch that contains changes, select the branch that contains the new
-   changes.
+.. code:: bash
 
-.. image:: ../../images/cilium-quayio-tag-4.png
-    :align: center
+    $ git commit --amend -s images/ -m "update cilium-{runtime,builder}"
```

Comment:
Nit: the `-m ...` option should not be necessary here since we are amending and the same commit message was already specified on the original commit.

```suggestion
    $ git commit --amend -s images/"
```